package.json

```diff
@@ -28,10 +28,10 @@
     "@aws-sdk/client-ssm": "^3.54.1",
     "@babel/core": "^7.4.4",
     "@babel/polyfill": "^7.4.4",
+    "@types/aws-lambda": "^8.10.92",
     "@types/config": "^0.0.39",
     "@types/tsscmp": "^1.0.0",
     "async-retry": "^1.2.3",
-    "aws-lambda": "^1.0.7",
```

Comment:
why are we removing aws-lambda here? is this part of node 18 upgrade? https://aws.amazon.com/blogs/compute/node-js-18-x-runtime-now-available-in-aws-lambda/